src/api/spec/controllers/webui/users/tokens_controller_spec.rb

```diff
@@ -95,6 +95,18 @@
         it { expect { subject }.not_to change(Token, :count) }
       end
     end
+
+    context 'type is service' do
+      context 'and the description is larger than 64 characters' do
+        render_views
+
+        let(:form_parameters) { { token: { type: 'service', description: 'Something' * 20 } } }
+
+        include_examples 'check for flashing an error'
```

Comment:
CI time is developer time. You are testing an activerecord validation with a click test.

Reply:
I'm testing that the form reacts correctly to validation errors. Before this change, the form crashed.

I will remove this test because it served it's purpose to confirm the bug presence and ensure the fix works.